### ISSUE TITLE
Test is not run in the MSVC configuration

### ISSUE THREAD

**Issue by msprotz** [project-everest/mitls-fstar#212]
https://github.com/project-everest/mitls-fstar/blob/master/src/windows/quiccrypto/makefile.vs#L22

this line doesn't work; I don't know how to do this in a visual studio makefile, short of calling into a .bat script; would anyone know how to make this work? @wintersteiger perhaps?

**Comment by wintersteiger** [comment]
Use `&` instead of `;`?

**Comment by msprotz** [comment]
```
D:\cygwin\home\protz\Code\mitls-fstar\src\windows\quiccrypto>nmake -f makefile.vs

Microsoft (R) Program Maintenance Utility Version 14.10.25017.0
Copyright (C) Microsoft Corporation.  All rights reserved.

        set PATH=..\kremlib;..\evercrypt;%PATH% & .\test.exe

D:\cygwin\home\protz\Code\mitls-fstar\src\windows\quiccrypto>
```

nothing seems to run... any clue how to diagnose this?

**Comment by wintersteiger** [comment]
Does it pop up any dialogs in the background saying that it didn't find DLL so and so?

**Comment by msprotz** [comment]
On my machine, when I copy/paste that line, it displays some output then fails with an illegal instruction error. Nothing shows up when this is executed via the Makefile...

**Comment by wintersteiger** [comment]
Right, it probably suppresses that for some reason. Maybe wrap I in `cmd /c "..."`?

**Comment by wintersteiger** [comment]
I can't run that makefile, it complains that it doesn't know how to produce .obj files... I assume that's expected?

**Comment by wintersteiger** [comment]
Aha! Yes, the `set` doesn't work properly when used like that, the call to `test.exe` will see the old path.

**Comment by wintersteiger** [comment]
Try this:

```
test: test.exe libquiccrypto.dll
	set PATH=..\kremlib;%PATH%
	echo PATH=%%PATH%%
	.\test.exe
```
